pylint/utils/utils.py

```diff
@@ -8,14 +8,41 @@
 import codecs
 import re
 import sys
+import textwrap
 import tokenize
 from os import linesep, listdir
 from os.path import basename, dirname, exists, isdir, join, normpath, splitext
 
 from astroid import Module, modutils
-from pylint.utils.constants import PY_EXTS
-from pylint.message.constants import MSG_TYPES, MSG_TYPES_LONG
-from pylint.utils.normalize_text import normalize_text
+
+from pylint.constants import MSG_TYPES, MSG_TYPES_LONG, PY_EXTS
+
+
+def normalize_text(text, line_len=80, indent=""):
+    """Wrap the text on the given line length."""
+    return "\n".join(
+        textwrap.wrap(
+            text, width=line_len, initial_indent=indent, subsequent_indent=indent
+        )
+    )
+
+
+CMPS = ["=", "-", "+"]
+
+# py3k has no more cmp builtin
+if sys.version_info >= (3, 0):
+
+    def cmp(a, b):  # pylint: disable=redefined-builtin
+        return (a > b) - (a < b)
+
+
+def diff_string(old, new):
```

Comment:
This function is not used anywhere during the test suite.We could check if it's used before refactoring it to merge the `sys.version_info >= (3, 0)` and `CMPS` inside it.

Reply:
Removed.